### ISSUE TITLE
Talk to multiple targets/devices simultaniously 

### ISSUE THREAD

**Issue by arthurlutz** [pycom/Pymakr#7]
It would be nice to be able to sync and execute on multiple targets that are available via FTP/telnet or serial.

**Comment by RalphHogenbirk** [comment]
Agreed, very nice to have. Added to my task list for later on. Not sure when I have time to do it though.
